client/components/gravatar/index.jsx

```diff
@@ -64,10 +73,18 @@ module.exports = React.createClass( {
 		}
 
 		const alt = this.props.alt || this.props.user.display_name;
-		const avatarURL = this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
+
+		const avatarURL = this.props.tempImage ||
+			this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
```

Comment:
Could we extract these props out of `props` at the top of the function? Like:

```js
const {
   alt,
   user,
   tempImage,
   size
   // etc...
} = this.props;
```

Also, shouldn't we write a new selector for getting the user's `avatar_URL` instead of accessing it directly like `user.avatar_URL`? :)